e2e/volumegroupsnapshot_base.go

```diff
@@ -199,6 +199,70 @@ func (v *volumeGroupSnapshotterBase) CreatePVCClones(
 	return pvcs, nil
 }
 
+func (v *volumeGroupSnapshotterBase) CreatePods(pvcs []*v1.PersistentVolumeClaim) ([]*v1.Pod, error) {
+	pods := make([]*v1.Pod, len(pvcs))
+	for i, p := range pvcs {
+		pods[i] = &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("pod-%d", i),
+				Namespace: p.Namespace,
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:    "container",
+						Image:   "quay.io/centos/centos:latest",
+						Command: []string{"/bin/sleep", "999999"},
```

Comment:
use `sleep infinity` instead?

Reply:
i think should be fine for now, i just copied it from other test file, planning for refractoring will take care of it in next PR.